src/permission/types.py

```diff
@@ -29,6 +31,21 @@ async def has_permission(db_session: database.DBSession, computing_id: str) -> b
 
         return False
 
+class ElectionOfficer:
+    @staticmethod
+    async def has_permission(db_session: database.DBSession, computing_id: str) -> bool:
+        """
+        An current elections officer has access to all elections, prior elections officers have no access.
+        """
+        officer_terms = await officers.crud.current_executive_team(db_session, True)
```

Comment:
`current_executive_team` has been replaced with `current_officers`. The behaviour *shooooould* be the same